Practise_20200314/Model/Entities/Brand.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Text;
+
+namespace Model.Entities
+{
+    public class Brand : INotifyPropertyChanged
```

Comment:
Actually a ViewModel should be used for such purposes.
Do we really need this here?